example/lib/pages/player/platform/platform_player_page.dart

```diff
@@ -0,0 +1,35 @@
+import 'package:bonfire/bonfire.dart';
+import 'package:example/pages/player/platform/simple_fox_player.dart';
+import 'package:flutter/widgets.dart';
+
+class PlatformPlayerPage extends StatelessWidget {
```

Comment:
Omg, loved it, I will definitely play around with it and help improving it in the near future.